core/extract/extractor.py

```diff
@@ -15,7 +15,7 @@
 from collections import OrderedDict
 from boto3 import client
 from tqdm import tqdm
-from core.util import audit_logs_parsing as audit_log_parser
+from core.replay.connections_parser import ConnectionLog
```

Comment:
Fix for using connectionLog from `connection_parser.py` and removed `audit_logs_parsing`